### DISCUSSION TITLE
Offer option to reset all changes made by Explorer patcher

### DISCUSSION THREAD

**Discussion by theking2** [valinet/ExplorerPatcher#1425] [Ideas]
# Problem
ExplorerPatcher occasionally causes a Windows 11 Explorer not to work properly anymore. Uninstall leaves all the settings, causing this, in place. 

# Idea
Offer the option to "reset" everything done by ExplorerPatcher on uninstall.

**Comment by ElfVierzehn** [comment]
yes. absolutely necessary. after a windows update my pc made today, nothing works anymore. just deleting everything completely breaks what it previously changed.
